cypress/integration/token-list/index.spec.ts

```diff
@@ -8,4 +14,35 @@ describe('Token list filter', () => {
     cy.get('#background-colors').should('exist');
     cy.get('#border-colors').should('not.exist');
   });
+
+  describe('Visual regression tests', () => {
+    it('Basic VRT', () => {
+      cy.log('[Applitools]: checking if eyes are enabled');
+      if (eyesAreEnabled()) {
+        cy.log('[Applitools]: Eyes are enabled, proceed with eyes check.');
+        const openParams: Partial<Eyes.Open.Options> = {
+          ...DEFAULT_OPEN_PARAMS,
+          testName: testSuiteName,
+        };
+
+        cy.log(`starting eyes with these params: ${openParams}`);
+
+        cy.eyesOpen(openParams);
+
+        const checkParams: Partial<Eyes.Check.Options> = {
+          ...DEFAULT_CHECK_PARAMS,
+          tag: `${testSuiteName}-basic-page-check`,
+        };
+
+        cy.log(`[Applitools]: Checking window with these params: ${checkParams}`);
+
+        cy.eyesCheckWindow(checkParams);
+
+        cy.eyesClose();
+      } else {
+        cy.log('[Applitools]: Eyes not enabled, skipping eyes check');
+      }
+      expect(true).to.equal(true);
```

Comment:
We need this so Cypress doesn't complain about the lack of an expect clause in the `it` block?